sdk/communication/communication-short-codes/test/public/utils/recordedClient.ts

```diff
@@ -5,28 +5,24 @@ import { Context } from "mocha";
 import * as dotenv from "dotenv";
 
 import {
-  env,
   Recorder,
-  record,
   RecorderEnvironmentSetup,
+  env,
+  isLiveMode,
   isPlaybackMode,
+  record,
```

Comment:
Would you mind filing an issue to track migrating tests to the new recorder?
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/MIGRATION.md